css/app.css

```diff
@@ -0,0 +1,400 @@
+/**
+ * Project: softexpo
+ * File: app.css
+ * Created: 2/14/23
+ * Author: Abdullah Al Mamun <mamun1214@gmail.com>
+ */
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+@import "common.css";
+@import "nav.css";
+
+#intro {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 10px;
+}
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 correct the margins and padding of this section to match the original design. In the about page the white navbar is overlapping the content of the about page, please 🙏🏽 fix it for a better UI.

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/219693829-f4cbc7c4-f6e2-4033-bbb3-ba8e3fd76f29.png) | ![image](https://user-images.githubusercontent.com/58642949/219693960-e03d52f4-c8cb-4dcc-b1a8-60fcc86141dd.png) |
| ![image](https://user-images.githubusercontent.com/58642949/219694868-b27d8520-9062-446d-8099-51a51b85d861.png) | ![image](https://user-images.githubusercontent.com/58642949/219694671-9c1b96a4-76c7-4c65-86d4-f003a1a386e4.png) |